packages/webapp/pages/account/notifications.tsx

```diff
@@ -398,6 +430,24 @@ const AccountNotificationsPage = (): ReactElement => {
             <Checkbox name="newsletter" checked disabled>
               Activity (mentions, replies, upvotes, etc.)
             </Checkbox>
+            <Checkbox
```

Comment:
Should we hide this behind the isStreaksEnabled? and also unsubscribe if user has enabled the notification but disables the reading streaks at all after

Reply:
👍

Reply:
Instead of hiding it, we could show it but as unchecked and disabled?
What do you think @rebelchris @nensidosari?

Reply:
If we show it it might be good to also show any info when u hover over that says they need to enable reading streaks if they want this setting. I think it's best to check with product/design for this. 

On my opinion to unblock and move forward I'd say let's hide, and then add as u suggested if product/design confirms as a later step - no need to do in this PR.

PS.
do we have any other checkbox here that stays disabled for some reason?

Reply:
I added the mutation for unsubscribing to the mutation for opt-out of streaks dde7ce7
![image](https://github.com/dailydotdev/apps/assets/1681525/536464a0-40a0-4acf-81cf-51837e137ee6)

Reply:
But I will also add a check in API just to be sure, you know, edge cases and all